neurots/generate/algorithms/tmdgrower.py

```diff
@@ -61,6 +65,12 @@ def __init__(
         self.apical_section = None
         self.apical_point_distance_from_soma = 0.0
         self.persistence_length = self.barcode.get_persistence_length()
+        if pia_direction is None:
+            self.pia_rotation = np.eye(3)
```

Comment:
As far as I can see, all the functions just do nothing when `pia_direction is None`, so why not just setting it to `None`?

Reply:
Indeed!